# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wdiff/strstr.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


Well - that's a new one. A program defines it's own prototype instead of
using the system headers.

 - Clifford

--- ./wdiff.c~	Sun Nov  6 08:57:23 1994
+++ ./wdiff.c	Thu Sep 24 20:20:46 1998
@@ -59,7 +59,7 @@
 # endif
 #endif
 
-char *strstr ();
+// char *strstr ();
 
 #if HAVE_TPUTS
 # if HAVE_TERMCAP_H
